cpp/daal/src/externals/service_service_mkl.h

```diff
@@ -37,26 +41,33 @@ namespace mkl
 {
 struct MklService
 {
-    static void * serv_malloc(size_t size, size_t alignment) { return fpk_serv_malloc(size, alignment); }
+    static void * serv_malloc(size_t size, size_t alignment) { return aligned_alloc(size, alignment); }
```

Comment:
I believe this thing already exists in the standard [C](https://en.cppreference.com/w/c/memory/aligned_alloc) & [C++](https://en.cppreference.com/w/cpp/memory/new/operator_new) libraries.